### DISCUSSION TITLE
Just a question for getUpdates()

### DISCUSSION THREAD

**Discussion by DullJZ** [eternnoir/pyTelegramBotAPI#1630] [Q&A]
In official docs, we should set a Webhook or use bot.getUpdates() to get the latest message manually to handle it.

Thanks to your efforts, bot.polling() working with some decorators seems to solve the problem. And it's needless to set a keep-running program to get new messages, and call the handler function by myself, do I say right?

**Comment by coder2020official** [comment]
With webhooks - you need to call process_new_updates function by your own, but polling does it for you.
